addons/website_forum/models/forum.py

```diff
@@ -161,6 +162,10 @@ def get_tags_first_char(self):
         tags = self.env['forum.tag'].search([('forum_id', '=', self.id), ('posts_count', '>', 0)])
         return sorted(set([tag.name[0].upper() for tag in tags if len(tag.name)]))
 
+    def _compute_website_url(self):
```

Comment:
This code is useless now, it was linked to the published mixin that you just removed